dask/async.py

```diff
@@ -426,69 +426,69 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
         result_flat = set([result])
     results = set(result_flat)
 
-    dsk = dsk.copy()
-    for f in start_cbs:
-        f(dsk)
-
-    dsk, dependencies = cull(dsk, list(results))
+    try:
+        dsk = dsk.copy()
+        for f in start_cbs:
+            f(dsk)
```

Comment:
Hrm, this seems to interfere with the `unpack_callbacks` bit.  `started_cbs` and `finish_cbs` don't necessarily align.

Do we know how expensive the extra function calls are that `unpack_callbacks` saves us from?